### ISSUE TITLE
Scottsdale buildings layer out of alignment

### ISSUE THREAD

**Issue by SeanWhitcomb** [facebook/Rapid#1008]
### Description

I'm using the City of Scottsdale, AZ Buildings layer, and all of the features are shifted southwest. All the imagery layers are lined up properly with the existing features on the map, so it's definitely the Rapid layer that's offset.

### Version

2.0.3

### What browser are you seeing the problem on? What version are you running?

Chrome v115.0

### The OS you're using

win

### Steps to reproduce

Add/Manage Datasets > Choose City of Scottsdale, AZ Buildings

### The browser URL at the time you encountered the bug

https://rapideditor.org/edit#background=Bing&datasets=602e0a13644a4daead72094e49d238ff&disable_features=boundaries&map=18.88/33.50496/-111.93704

### The auto-detected useragent string for your browser (leave blank if you're manually filling this form out)

Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36

**Comment by Bonkles** [comment]
Checking the alignment in Scottsdale- here's a screenshot taken at zoom 17: 
![image](https://github.com/facebook/Rapid/assets/1887955/99bb5900-a9d5-44fc-9f0b-95d7f817548f)

Zooming in, you can see that every building is just a bit south of where it should be, resulting in quite a few issues. 

![image](https://github.com/facebook/Rapid/assets/1887955/00bcaa0d-b1e3-4ecd-b183-a9ab80091670)

I don't think we can ever be fully free of shift/drift of features as compared to imagery. does this discrepancy look normal to you, too?

**Comment by bhousel** [comment]
Yeah - #270 would let people adjust the alignment of the data.. We probably need to add some "data settings" screen that would let people do this.

Here is my professionally-designed mockup of how it might look:

<img width="707" alt="Screenshot 2023-08-11 at 5 36 45 PM" src="https://github.com/facebook/Rapid/assets/38784/c10aae71-df61-4fd7-9870-39b77cfe67b4">

**Comment by SeanWhitcomb** [comment]
It would be really helpful to be able to adjust the data layer. Imagery offset wouldn’t help in this case since it’s the data that is off. Sean Whitcomb Prounouns: he/him/hisMesa Community CollegeMARICOPA COMMUNITY COLLEGESResidential Faculty | Life ScienceDirector | MCC Arboretum1833 West Southern Avenue, Mesa AZ ***@***.***://www.mesacc.edu/480-461-7117On Aug 11, 2023, at 2:47 PM, Bryan Housel ***@***.***> wrote:﻿
Yeah - #270 would let people adjust the alignment of the data.. We probably need to add some "data settings" screen that would let people do this.
Here is my professionally-designed mockup of how it might look:
